commands/builds/create.js

```diff
@@ -8,84 +8,93 @@ let path = require('path')
 let exec = require('child_process').execSync
 let nodeTar = require('../../lib/node_tar')
 
-function compressSource (context, cwd, tempFile, cb) {
-  var tar = context.flags['tar'] || 'tar'
-  let tarVersion = exec(tar + ' --version').toString()
+function compressSource (context, cwd, tempFile) {
+  return new Promise(function (resolve, reject) {
+    var tar = context.flags['tar'] || 'tar'
+    let tarVersion = ''
 
-  if (tarVersion.match(/GNU tar/)) {
-    let includeVcsIgnore = context.flags['include-vcs-ignore']
-    let command = tar + ' cz -C ' + cwd + ' --exclude .git --exclude .gitmodules .'
-
-    if (!includeVcsIgnore) {
-      command += ' --exclude-vcs-ignores'
+    try {
+      tarVersion = exec(tar + ' --version').toString()
+    } catch (err) {
+      // tar was not found on the system
```

Comment:
You're absolutely right. I've updated the catch to rescue only if the command is missing.

I tried using `spawnAsync` instead of `execAsync` so it wouldn't throw. But for some reason, it doesn't initialize the environment similarly, and tar isn't found with it.